storm/src/main/storm/mesos/MesosNimbus.java

```diff
@@ -200,8 +206,9 @@ void initializeMesosStormConf(Map conf, String localDir) {
       throw new RuntimeException(String.format("Encountered IOException while setting up LocalState at %s : %s", localDir, exp));
     }
 
-    _allowedHosts = listIntoSet((List<String>) conf.get(CONF_MESOS_ALLOWED_HOSTS));
-    _disallowedHosts = listIntoSet((List<String>) conf.get(CONF_MESOS_DISALLOWED_HOSTS));
```

Comment:
Ok, so you've removed these 2 features (`mesos.allowed.hosts` & `mesos.disallowed.hosts`).  Please add to the PR description an explicit statement about this feature being removed, including the justification for removing these existing features and how the new feature can be used instead.  Also please update the README references to this feature to state how the new feature can be used to replicate the same behavior.  Which reminds me that we still need docs about using the new features, per [this earlier comment](https://github.com/mesos/storm/pull/196#discussion_r109566492) that I made.